web/modules/custom/dpl_reservations/src/Plugin/Block/ReservationListBlock.php

```diff
@@ -0,0 +1,200 @@
+          $this->t('Cancel reservations', [], $context),
+        ],
+      ],
+      'delete-reservation-modal-delete-question-text' => $this->t('Do you want to cancel your reservation?', [], $context),
+      'delete-reservation-modal-delete-button-text' => $this->t('Cancel reservation', [], $context),
+      'delete-reservation-modal-not-regrettable-text' => $this->t('You cannot regret this action', [], $context),
+      'delete-reservation-modal-close-modal-text' => $this->t('Close delete reservation modal', [], $context),
+      'delete-reservation-modal-aria-description-text' => $this->t('This button opens a modal that covers the entire page and contains the possibility to delete a selected reservation, or multiple selected reservations', [], $context),
+      'reservation-list-on-hold-aria-text' => $this->t('Reservations have been paused in the following time span:', [], $context),
+      'reservation-list-pause-reservation-aria-modal-text' => $this->t('This button opens a modal that covers the entire page and contains the possibility to pause physical reservations', [], $context),
+      'pause-reservation-modal-aria-description-text' => $this->t('This modal makes it possible to pause your physical reservations', [], $context),
+      'pause-reservation-modal-header-text' => $this->t('Pause reservations on physical items', [], $context),
+      'pause-reservation-modal-body-text' => $this->t('Pause your reservations early, since reservations that are already being processed, will not be paused.', [], $context),
+      'pause-reservation-modal-close-modal-text' => $this->t('Close pause reservations modal', [], $context),
+      'date-inputs-start-date-label-text' => $this->t('Start date', [], $context),
+      'material-details-close-modal-aria-label-text' => $this->t("Close material details modal", [], $contextAria),
+      'date-inputs-end-date-label-text' => $this->t('End date', [], $context),
+      'pause-reservation-modal-below-inputs-text-text' => $this->t('Pause reservation below inputs text', [], $context),
+      'pause-reservation-modal-link-text' => $this->t('Read more about pausing reservertions and what that means here', [], $context),
+      'pause-reservation-modal-save-button-label-text' => $this->t('Save', [], $context),
+      'one-month-text' => $this->t('1 month', [], $context),
+      'two-months-text' => $this->t('2 months', [], $context),
+      'three-months-text' => $this->t('3 months', [], $context),
+      'six-months-text' => $this->t('6 months', [], $context),
+      'one-year-text' => $this->t('1 year', [], $context),
+      'list-details-nothing-selected-label-text' => $this->t('Pick', [], $context),
+      'show-more-text' => $this->t('show more', [], $context),
+      'result-pager-status-text' => $this->t('Showing @itemsShown out of @hitcount results', [], $context),
+      'reservation-list-status-icon-ready-for-pickup-aria-label-text' => $this->t('This material is ready for pickup', [], $context),
```

Comment:
Consider if this and other places should use the `$contextAria`.

As far as I can see you define the context but only use it once. For whatever reason it is not used for this prop which is explicitly referring aria.

Reply:
Fixed and reused in more places in this file.